e2e/playwright/testing-segment-overlays.spec.ts

```diff
@@ -90,15 +92,12 @@ test.describe('Testing segment overlays', () => {
         await expect(
           page.getByTestId('cmd-bar-arg-value').getByRole('textbox')
         ).toBeFocused()
+        await page.waitForTimeout(500)
         await page
           .getByRole('button', {
             name: 'arrow right Continue',
           })
           .click()
-        await expect(page.locator('.cm-content')).toContainText(expectFinal)
-        await editor.expectEditor.toContain(expectFinal, {
-          shouldNormalise: true,
-        })
         await editor.expectEditor.toContain(expectFinal, {
           shouldNormalise: true,
         })
```

Comment:
This was very wtf, asserting the same thing 3 times in a row for some unknown reason.